index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css ">
+        <script src="https://kit.fontawesome.com/5116129a4d.js"
+            crossorigin="anonymous"></script>
+        <title>AALA</title>
+    </head>
+
+    <body>
+
+        <section class="sec1">
+            <div class="desktopdarkheader">
+                <div class="dark-items">
+                    <ul>
+                        <li><i class="fa-brands fa-facebook-f"></i></li>
+                        <li><img src="images/twitter.png" alt=""></li>
+                        <li>English</li>
+                        <li>My Page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+            </div>
+            <div class="desktoplighterheader">
+                <div class="desktopheader">AALA 2022</div>
+
+                <nav class="references">
+                    <a href="about.html" class="about">About</a>
+                    <a href="#mainprogramme" class="programme">Program</a>
+                    <a href="">Join</a>
+                    <a href="#sec4">Sponsor</a>
+                    <a href="">News</a>
+                </nav>
+            </div>
+
+
+
+            <div class="container">
+                <div id="container">
+                    <button class="btn"><i class="fa-solid fa-bars navbtn"></i></button>
+                    <div class="txt">
+                        <h1 class="text">Hello! Welcome</h1>
+                    </div>
+                    <div id="description">
+                        <h1 class="description">Annual Art And Literary
+                            Appreciation 2022!</h1>
```

Comment:
Kindly note that there should only be one `h1` tag per page to follow the best practices, so kindly remove the second `h1` tag 🤗